pkg/controller/sparkapplication/controller.go

```diff
@@ -122,7 +122,7 @@ func newSparkApplicationController(
 	}
 
 	if metricsConfig != nil {
-		controller.metrics = newSparkAppMetrics(metricsConfig.MetricsPrefix, metricsConfig.MetricsLabels)
+		controller.metrics = newSparkAppMetrics(metricsConfig.MetricsPrefix, metricsConfig.MetricsLabels, metricsConfig.MetricsJobStartLatencyBuckets)
```

Comment:
Looks like we really should just pass in the `metricsConfig`.

Reply:
Sounds good, will fix it in the next commit